src/everest/detached/jobs/everserver.py

```diff
@@ -447,17 +503,21 @@ def main():
 
         assert experiment_status is not None
         status, message = _get_optimization_status(experiment_status, shared_data)
+
         if status != ServerStatus.completed:
+            send_end_event(shared_data, True, message)
```

Comment:
It seems there are multiple `EndEvent` messages being sent once by the run model and then also by the server is that not a problem?

Reply:
It should be fine, as the monitor shuts down after receiving one, but I will review how we send the end events.